frontend/src/components/panels/index.js

```diff
@@ -0,0 +1,9 @@
+import WelcomePanel from './home_panel/WelcomePanel';
+import ProjectListPanel from './project_list_panel/ProjectListPanel';
+import ProjectFiltersPanel from './project_filters_panel/ProjectFiltersPanel';
```

Comment:
I think you could remove some of the depth to the project repo by structuring this as follows

```
./ProjectFilterPanel/index.js <- actually define and export the core component
./ProjectFilterPanel/ChildComponent.js <- a component defined but only used in ProjectFilterPanel component
```

This would simplify your imports to just `import ProjectFilterPanel from './ProjectFilterPanel';`